bin/ch/WScriptJsrt.cpp

```diff
@@ -725,7 +725,7 @@ bool WScriptJsrt::InstallObjectsOnObject(JsValueRef object, const char* name, Js
     memcpy(path, msg, (size_t)str_len);          \
     path[str_len] = char(0)
 
-void GetBinaryLocation(char *path, const uint32_t size)
+void GetBinaryLocation(char *path, const unsigned size)
```

Comment:
Yes, sizeof returns a size_t. If the buffer size is guaranteed to be 32 bits or smaller on all platforms then there's no problem.